src/uu/ls/src/ls.rs

```diff
@@ -1980,16 +2054,33 @@ fn display_file_name(
     path: &PathData,
     config: &Config,
     prefix_context: Option<usize>,
+    out: &mut BufWriter<Stdout>,
 ) -> Option<Cell> {
     // This is our return value. We start by `&path.display_name` and modify it along the way.
     let mut name = escape_name(&path.display_name, &config.quoting_style);
 
     #[cfg(unix)]
     {
         if config.format != Format::Long && config.inode {
-            name = path.md().map_or_else(|| "?".to_string(), get_inode) + " " + &name;
+            name = match path.md() {
+                Some(md) => get_inode(md),
+                None => {
+                    let _ = out.flush();
+                    show!(LsError::IOErrorContext(
+                        std::io::Error::new(ErrorKind::NotFound, "NotFound"),
+                        path.display_name.to_owned()
+                    ));
+                    "?".to_string()
+                }
+            } + " "
+                + &name;
```

Comment:
Please run `cargo fmt`

Reply:
I think `cargo fmt` is what is doing this!  I run rust fmt and cargo fmt before each commit.